--- libpfm-4.7.0/config.mk~	2016-02-11 17:11:30.000000000 +0000
+++ libpfm-4.7.0/config.mk	2018-09-08 23:31:34.687561366 +0000
@@ -198,19 +198,19 @@
 # -Wextra: to enable extra compiler sanity checks (e.g., signed vs. unsigned)
 # -Wno-unused-parameter: to avoid warnings on unused foo(void *this) parameter
 #
-DBG?=-g -Wall -Werror -Wextra -Wno-unused-parameter
+DBG?=-g -Wall -Wextra -Wno-unused-parameter
 
 ifeq ($(SYS),Darwin)
 # older gcc-4.2 does not like -Wextra and some of our initialization code
 # Xcode uses a gcc version which is too old for some static initializers
 CC=clang
-DBG?=-g -Wall -Werror
+DBG?=-g -Wall
 LDCONFIG=true
 endif
 
 ifeq ($(SYS),FreeBSD)
 # gcc-4.2 does not like -Wextra and some of our initialization code
-DBG=-g -Wall -Werror
+DBG=-g -Wall
 endif
 
 CFLAGS+=$(OPTIM) $(DBG) -I$(SYSINCDIR) -I$(PFMINCDIR)
--- libpfm-4.7.0/config.mk~	2018-09-08 23:31:34.000000000 +0000
+++ libpfm-4.7.0/config.mk	2018-09-08 23:34:36.299558838 +0000
@@ -116,7 +116,7 @@
 #
 # Where should things (lib, headers, man) go in the end.
 #
-PREFIX=/usr/local
+PREFIX=/usr/
 LIBDIR=$(PREFIX)/lib
 INCDIR=$(PREFIX)/include
 MANDIR=$(PREFIX)/share/man
@@ -190,7 +190,7 @@
 CC?=gcc
 LIBS=
 INSTALL=install
-LDCONFIG=ldconfig
+LDCONFIG=true
 LN?=ln -sf
 PFMINCDIR=$(TOPDIR)/include
 PFMLIBDIR=$(TOPDIR)/lib
